apps/meteor/ee/app/livechat-enterprise/server/api/units.ts

```diff
@@ -122,3 +77,46 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'livechat/units/:unitId/departments',
+	{ authRequired: true },
+	{
+		async get() {
+			const { offset, count } = this.getPaginationItems();
+			const { unitId } = this.urlParams;
+
+			const { departments, total } = await findAllDepartmentsByUnit(unitId, offset, count);
+
+			return API.v1.success({
+				departments,
+				count: departments.length,
+				offset,
+				total,
+			});
+		},
+	},
+);
+
+API.v1.addRoute(
+	'livechat/units/:unitId/departments/available',
+	{ authRequired: true },
+	{
+		get() {
+			const { offset, count } = this.getPaginationItems();
+			const { unitId } = this.urlParams;
+			const { text, onlyMyDepartments } = this.queryParams;
+
+			const { departments, total } = Promise.await(
+				findAllDepartmentsAvailable(this.userId, unitId, offset, count, text, onlyMyDepartments === 'true'),
+			);
+
+			return API.v1.success({
+				departments,
+				count: departments.length,
+				offset,
+				total,
+			});
+		},
```

Comment:
```suggestion
		async get() {
			const { offset, count } = this.getPaginationItems();
			const { unitId } = this.urlParams;
			const { text, onlyMyDepartments } = this.queryParams;

			const { departments, total } = await findAllDepartmentsAvailable(
				this.userId,
				unitId,
				offset,
				count,
				text,
				onlyMyDepartments === 'true',
			);

			return API.v1.success({
				departments,
				count: departments.length,
				offset,
				total,
			});
		},
```
Avoiding promise.await

Reply:
Looks like the tests were being ignored 👀